src/docs/010_manual/60_further_reading.adoc

```diff
@@ -6,102 +6,100 @@ ifndef::imagesdir[:imagesdir: ../images]
 
 include::../_feedback.adoc[]
 
```

Comment:
Add a heading so the page doesn't start with a single sentence:

== Introduction

Reply:
Useful Resources is the heading.. I could rephrase it as just resources though

Reply:
Useful Resources is the page heading and it's fine as it is. I mean the first heading on the actual page. It looks a bit weird to start with a sentence then have a whole bunch of other headings on the page.